esp/logging/logginglib/logthread.cpp

```diff
@@ -155,7 +155,10 @@ bool CLogThread::queueLog(IEspUpdateLogRequest* logRequest)
 
 bool CLogThread::queueLog(IEspUpdateLogRequestWrap* logRequest)
 {
+    unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
```

Comment:
same comments as above regarding the starttime, and the conditional on loglevel + DBGLOG